### PR TITLE
feat(sui-test): add ci mode and update cypress version

### PR DESCRIPTION
**Pull request by jcger**
BREAKING CHANGE:
update cypress version

### CODE REVIEW

File: packages/sui-test/bin/sui-test-e2e.js [hunk 1]
```diff
@@ -86,6 +88,11 @@ if (screenshotsOnError) {
   cypressConfig.screenshotsFolder = SCREENSHOTS_FOLDER
 }
 
+if (ci) {
```
Comment (midudev) [inline packages/sui-test/bin/sui-test-e2e.js:91]: 😮 mola.

File: packages/sui-test/Dockerfile [hunk 2]
```diff
@@ -2,8 +2,6 @@ FROM cypress/base
 
 WORKDIR /usr/src
 
-ARG CYPRESS_VERSION=3.8.3
-
-RUN npm install --force-only @s-ui/test cypress@$CYPRESS_VERSION
+RUN npm install --force-only @s-ui/test
```
Comment (carlosvillu) [inline packages/sui-test/Dockerfile:5]: Why are you remove the installation of Cypress ?!

Comment (jcger) [reply packages/sui-test/Dockerfile:5]: didn't catch at first that this is to avoid lazy load the package, isn't it? gonna roll it back

Comment (tomcask) [reply packages/sui-test/Dockerfile]: If we remove cypress, where will it be then? In the base image of docker there is no cypress.

In the [documentation](https://hub.docker.com/r/cypress/base/) it recommends using this:

`FROM cypress/base:10
RUN npm install --save-dev cypress
RUN $(npm bin)/cypress verify
RUN $(npm bin)/cypress run`

File: packages/sui-test/README.md [hunk 3]
```diff
@@ -135,6 +135,7 @@ Then you can use in your specs `cy.login()`
     -U, --userAgentAppend <userAgentAppend>  Append string to UserAgent header.
     -UA, --userAgent <userAgent>             Overwrite string to UserAgent header.
     -G, --gui                                Run the tests in GUI mode.
+    -C, --ci                                 CI Mode, reduces memory consumption
```
Comment (nucliweb) [inline packages/sui-test/README.md:138]: 👏🏼

File: packages/sui-test/Dockerfile (hunk 2 above)
Comment (jcger) [reply packages/sui-test/Dockerfile]: it's being lazy loaded by the sui-test itself https://github.com/SUI-Components/sui/blob/master/packages/sui-test/bin/sui-test-e2e.js#L89

Comment (tomcask) [reply packages/sui-test/Dockerfile]: ok!!!

Comment (tomcask) [reply packages/sui-test/Dockerfile]: 🎉

**Pull request merged**